src/data/services/EnterpriseAccessApiService.js

```diff
@@ -159,6 +159,17 @@ class EnterpriseAccessApiService {
     return EnterpriseAccessApiService.apiClient().get(url);
   }
 
+  /**
+   * Cancel content assignments for a specific AssignmentConfiguration.
+   */
+  static cancelContentAssignments(assignmentConfigurationUUID, uuids) {
```

Comment:
nit: perhaps `assignmentUuids` for more explicit/descriptive variable names ("what uuids are these referring to?").